src/main/java/io/scif/formats/TIFFFormat.java

```diff
@@ -443,8 +443,20 @@ protected void initMetadata(final Metadata meta, final SCIFIOConfig config)
 				parseCommentGeneric(meta, comment);
 			}
 
-			// check for another file with the same name
+			// map third dimension to Z axis if no comments are available
```

Comment:
Change this to `map third dimension to unknown axis if no metadata is available`